check-log/check-log.go

```diff
@@ -154,6 +164,9 @@ func (opts *logOpts) searchLog(logFile string) (int64, int64, string, error) {
 
 	f, err := os.Open(logFile)
 	if err != nil {
```

Comment:
I agree. I respected excessively original behavior by mistake. I fixed it d2f7981